tests/dynamic_data/publications/layer_raster_contrast/contrast_test.py

```diff
@@ -15,6 +15,7 @@
     'tif_byte': {},
     'tif_float': {},
     'tif_byte_nodata_high': {},  # nodata > max value
+    'tif_byte_nodata_low': {},  # nodata < min value
```

Comment:
It would be nice to test min_value, max_value and nodata value for these cases during test.